clang/test/ParserSYCL/unique_stable_name.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -Wno-unused %s
+
+namespace NS{};
+
+void f(int var) {
+  // expected-error@+1{{expected '(' after '__builtin_unique_stable_name'}}
+  __builtin_unique_stable_name int;
+  // expected-error@+1{{expected '(' after '__builtin_unique_stable_name'}}
+  __builtin_unique_stable_name {int};
+
```

Comment:
You should also have some tests for the closing paren. e.g.,
```
__builtin_unique_stable_name(int; // Missing paren before semicolon
__builtin_unique_stable_name(int, float); // Missing paren before comma
```

Reply:
Added these tests